### PR TITLE
Error logs for AppsFlyer if missing networkUserID

### PR DESCRIPTION
**Pull request by vegaro**
- Added two error logs
- Fixed typo in the NS_SWIFT_NAME. This will break compilation but there's no way to deprecate it.

### CODE REVIEW

File: Purchases/Public/RCPurchases.m [hunk 1]
```diff
@@ -295,6 +295,13 @@ - (void)postAttributionData:(NSDictionary *)data
                fromNetwork:(RCAttributionNetwork)network
           forNetworkUserId:(NSString * _Nullable)networkUserId
 {
+    if (network == RCAttributionNetworkAppsFlyer && networkUserId == nil) {
+        if (data[@"rc_appsflyer_id"]) {
```
Comment (jeiting) [inline Purchases/Public/RCPurchases.m]: Shouldn't this live outside of the if?

File: Purchases/Public/RCPurchases.m [hunk 2]
```diff
@@ -295,6 +295,13 @@ - (void)postAttributionData:(NSDictionary *)data
...
+            RCErrorLog(@"⚠️ The parameter key rc_appsflyer_id is deprecated. Pass networkUserId to addAttribution instead. ⚠️");
+        } else {
+            RCErrorLog(@"⚠️ The parameter networkUserId is REQUIRED for AppsFlyer. ⚠️");
```
Comment (jeiting) [inline Purchases/Public/RCPurchases.m]: This should always be logged inside this if.

File: Purchases/Public/RCPurchases.m (hunk 1 above)
Comment (vegaro) [reply Purchases/Public/RCPurchases.m]: oh yeah makes sense hehe

**Comment by vegaro** [comment]
@jeiting updated

**Pull request merged**